src/content/development/contribution-guide/_index.en.md

```diff
@@ -10,23 +10,38 @@ in order for their contributions to the project to be addressed in a timely mann
 
 ## Project Resources
 
-To be expanded
+Submariner uses [GitHub Projects] to manage releases. Read [Tracking Progress on your Project Board] to learn more on how to work with
+projects.
 
 ### Backlog Board
 
-To be expanded
+[The backlog board] hosts issues or features that are intended to be worked upon. While each issue is opened in
```

Comment:
```suggestion
[The backlog board] hosts issues or features that are intended to be worked on eventually. While each issue is opened in
```

This is better grammer, but I don't know if the content makes sense. I think all open Issues are intended to be worked on, not just ones on the backlog.

Maybe something like

```suggestion
[The backlog board] hosts issues or features that are not a part of a particular release board. While each issue is opened in
```

Reply:
> This is better grammer, but I don't know if the content makes sense. I think all open Issues are intended to be worked on, not just ones on the backlog.
> 
> Maybe something like

All issues opened as supposed to be in `Backlog` board.